src/device-registry/models/Airqloud.js

```
@@ -224,12 +224,13 @@
   try {
     const inclusionProjection = constants.AIRQLOUDS_INCLUSION_PROJECTION;
     const exclusionProjection = constants.AIRQLOUDS_EXCLUSION_PROJECTION(
-      filter.category ? filter.category : "none"
+      filter.path ? filter.path : "none"
     );
 
-    if (!isEmpty(filter.category)) {
-      delete filter.category;
+    if (!isEmpty(filter.path)) {
+      delete filter.path;
     }
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)